terraform/tfstate.json

```diff
@@ -17,7 +17,7 @@
       "mode": "data",
       "type": "github_users",
       "name": "users",
-      "provider": "provider[\"registry.terraform.io/integrations/github\"]",
+      "provider": "provider[\"registry.opentofu.org/integrations/github\"]",
```

Comment:
I'm not sure about this. I suspect it's because you used tofu locally. The github action is going to switch this back resulting in a fair amount of noise.

Reply:
yeah, github actions should switch it back.

Reply:
also the `terraform_version`